README.md

```diff
@@ -32,6 +32,19 @@ then
   slug: "cats",
   }]}
 
+### /api/articles/:article_id
+
+- responds with a specific article based on the id provided in the endpoint
+- example response:
+  {"article_id": 4,
+  "title": "Student SUES Mitch!",
+  "topic": "mitch",
+  "author": "rogersop",
+  "body": "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+  "created_at": "2020-05-06T01:14:00.000Z",
+  "votes": 0,
+  "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"}
```

Comment:
This shouldn't be in your readme, you have an endpoints.json file **specifically** for this information